buddyboss/buddyboss-platform
# #1355 - Fix PHP Error: Fix grammaticle group label text

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1355

### How to test the changes in this Pull Request:

1. When creating a group type label for any label that should be preceded by "a," groups will display "You're an" instead of "You're a."
2. Go to your -> site/groups/
3. See label

### Other information:
